redisinsight/ui/src/components/query-card/QueryCardHeader/QueryCardHeader.tsx

```diff
@@ -318,6 +344,26 @@ const QueryCardHeader = (props: Props) => {
             </EuiToolTip>
           )}
         </EuiFlexItem>
+        <EuiFlexItem
+          grow={false}
+          className={cx(styles.buttonIcon, styles.viewTypeIcon)}
+          onClick={onDropDownViewClick}
+        >
+          {isOpen && profileOptions.length > 1 && canCommandProfile && !summaryText && (
```

Comment:
can be profileOptions length less than 2?

Reply:
Removed the check. `profileOptions` will always have 2 elements (`PROFILE` and `EXPLAIN`).